model/feature_flags.go

```diff
@@ -81,6 +81,8 @@ type FeatureFlags struct {
 	ThreadsEverywhere bool
 
 	GlobalDrafts bool
+
+	UrlMappingForCWS bool
```

Comment:
This can probably be removed. You're not checking the value of it above - since this side of things would rely on the CWS passing the parameter, i think configuring it off on the CWS side would suffice